symengine/printer.cpp

```diff
@@ -124,6 +124,47 @@ void StrPrinter::bvisit(const Interval &x)
     str_ = s.str();
 }
 
+void StrPrinter::bvisit(const BooleanAtom &x)
+{
+    if (x.get_val()) {
+        str_ = "True";
+    } else {
+        str_ = "False";
+    }
+}
+
+void StrPrinter::bvisit(const Contains &x)
+{
+    std::ostringstream s;
+    s << "Contains(" << apply(x.get_expr()) << ", " << apply(x.get_set())
+      << ")";
+    // s << apply(x.get_expr()) << " " << "∈" << " " << apply(x.get_set());
```

Comment:
That, what is the use of this commented code ?

On Sun, Jul 31, 2016 at 5:35 AM, Isuru Fernando <notifications@github.com>
wrote:

> In symengine/printer.cpp
> <https://github.com/symengine/symengine/pull/1050#discussion_r72895929>:
>
> > @@ -124,6 +124,47 @@ void StrPrinter::bvisit(const Interval &x)
> >      str_ = s.str();
> >  }
> >
> > +void StrPrinter::bvisit(const BooleanAtom &x)
> > +{
> > +    if (x.get_val()) {
> > +        str_ = "True";
> > +    } else {
> > +        str_ = "False";
> > +    }
> > +}
> > +
> > +void StrPrinter::bvisit(const Contains &x)
> > +{
> > +    std::ostringstream s;
> > +    s << "Contains(" << apply(x.get_expr()) << ", " << apply(x.get_set())
> > +      << ")";
> > +    // s << apply(x.get_expr()) << " " << "∈" << " " << apply(x.get_set());
>
> I'm not sure I understand what you are saying
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/symengine/symengine/pull/1050/files/3d285de10940ecb96c91467b1f73c16914d625b5#r72895929>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AI0uixaEHzzzS8THB9dvNFYepXjh0EXtks5qa-bBgaJpZM4JYwpl>
> .
>



-- 
Nishant Nikhil
Sophomore
Department of Mathematics
IIT Kharagpur
| Mobile: +91 9933979842 <%2B91%2099339798428>
| Email: nishantnikhil@iitkgp.ac.in
<http://www.krssg.in/>